hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
```

Comment:
You provide two features to highlight the signal components: different marker styles and the horizontal line. So, what about controlling them with the following keywords instead: `n_pcs=None`, `highlight_pcs=True`, `hline=True`?